lib/puppet/provider/iis_site/webadministration.rb

```diff
@@ -0,0 +1,182 @@
+require_relative '../../../puppet_x/puppetlabs/iis/powershell_manager'
```

Comment:
We may want to avoid `require_relative` (https://tickets.puppetlabs.com/browse/FM-5325), I believe to avoid breaking compat w/open source masters on distros. That is, if we want to worry about that scenario here. Instead we could use the (admittedly less concise) `require File.join(File.dirname(__FILE__), '../../../puppet_x/puppetlabs/iis/powershell_manager')` ?